pull_request_template.md

```diff
@@ -1,7 +1,11 @@
+## Reminders
+- [ ] Did I review my own code.
+- [ ] Is there only 1 commit.
+
 ## Checklist before requesting a review
 - [ ] I have performed a self-review of my code
 - [ ] I have added thorough tests. (If tests are not applicable, explain why in the next section.)
 
```

Comment:
Let's do this:

```suggestion
## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have added thorough tests. (If tests are not applicable, explain why in the next section.)
- [ ] Is there only 1 commit.
```

Reply:
You can remove the reminders section afterwards

Reply:
Also squash your commits

Reply:
I removed it and squash the commit 👌

Reply:
I still see 6 commits

Reply:
sorry now it should be good, i got this error 'Updates were rejected because the tip of your current branch is behind'. I git push --force, it should be good now